assets/javascripts/discourse/pre-initializers/extend-category-for-solved.js.es6

```diff
@@ -1,22 +1,25 @@
-import property from "ember-addons/ember-computed-decorators";
 import Category from "discourse/models/category";
 
 export default {
   name: "extend-category-for-solved",
+
   before: "inject-discourse-objects",
+
   initialize() {
     Category.reopen({
-      @property("custom_fields.enable_accepted_answers")
-      enable_accepted_answers: {
-        get(enableField) {
-          return enableField === "true";
-        },
-        set(value) {
-          value = value ? "true" : "false";
-          this.set("custom_fields.enable_accepted_answers", value);
-          return value;
+      enable_accepted_answers: Ember.computed(
+        "custom_fields.enable_accepted_answers",
+        {
+          get(enableField) {
+            return enableField === "true";
+          },
+          set(value) {
+            value = value ? "true" : "false";
+            this.set("custom_fields.enable_accepted_answers", value);
```

Comment:
I know you didn't write this code, but why is `enable_accepted_answers` a string instead of a boolean?

Reply:
I dont know, but I  also thought it was a smell and saw https://github.com/discourse/discourse-solved/blob/master/plugin.rb#L412 and decided to not change it_